openstack/neutron/files/mitaka/neutron+plugins+ml2+drivers+linuxbridge+agent+linuxbridge_neutron_agent.py

```diff
@@ -463,9 +463,13 @@ def _add_tap_interface(self, network_id, network_type, physical_network,
                 return False
             #self.ensure_tap_mtu(tap_device_name, phy_dev_name)
 
-        # Do mess with any devices encountered
-        ip_lib.IPDevice(tap_device_name).link.set_master(bridge_name,
-            mtu_size=cfg.CONF.network_device_mtu, down=False)
+        # Set only some properties on links that are not governed by neutron
+        network_link = ip_lib.IPDevice(tap_device_name).link
+        if device_owner.startswith(constants.DEVICE_OWNER_PREFIXES) or physical_network in self.bridge_mappings:
+            network_link.set_master(bridge_name, mtu_size=cfg.CONF.network_device_mtu, down=False)
+        else:
+            network_link.set_params(up=True, mtu=cfg.CONF.network_device_mtu)
```

Comment:
@mkalcok-cisco those names could've been consistent with the set_master. Also, master could be one of the params for set_params, and so we'd call the same function in both cases.